rfc077-mode-agnostic-data-collection.md

```diff
@@ -525,6 +445,7 @@ The Run End Schema will be used by Chef Client to notify the data collection ser
         "expanded_run_list",
         "message_type",
         "message_version",
+        "node",
```

Comment:
yeah at some point we probably better want to filter or manage the contents of the node object itself, but we likely want to do that as a core feature